security/advancedtls/crl.go

```diff
@@ -136,7 +137,7 @@ func x509NameHash(r pkix.RDNSequence) string {
 	return fmt.Sprintf("%08x", fileHash)
 }
 
-// CheckRevocation checks the connection for revoked certificates based on RFC5280
+// CheckRevocation checks the connection for revoked certificates based on RFC5280.
 // This implementation has the following major limitations:
 // 	Indirect CRL files are not supported
 // 	File based lookup only using the X509_LOOKUP_hash_dir format
```

Comment:
English looks a little off here.  One suggestion: "Only the X509_LOOKUP_hash_dir format can be used with file-based lookup".  Or "File based lookup only supports the X509_LOOKUP_hash_dir format".